client/package.json

```diff
@@ -14,6 +14,7 @@
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.1.0",
     "react-test-renderer": "^16.9.0",
+    "react-textarea-autosize": "^7.1.0",
```

Comment:
why additional package when we have semantic text area?
https://react.semantic-ui.com/addons/text-area/

Reply:
Haha! :smile: 
![image](https://user-images.githubusercontent.com/26798929/63767385-1e232400-c8eb-11e9-9c92-bbf9a69e5a23.png)